docs/ovndb_ha.md

```diff
@@ -0,0 +1,56 @@
+# OVN Northbound/Southbound DB HA using Corosync/Pacemaker
+
+The following document captures how to start OVN DBs in Active/Standby
+HA mode using Corosync and Pacemaker. This HA mode is supported by OVS
```

Comment:
It would be good to have links to the corosync and pacemaker projects.

Reply:
Done.